src/commands/pull-requests.ts

```diff
@@ -282,17 +282,18 @@ export class CreatePullRequest extends PullRequestCommand {
     uri: vscode.Uri
   ): Promise<string | undefined> {
     // sort default branch up
+    const currentBranch = await this.git.getCurrentBranch(uri)
     const picks = (await this.git.getRemoteBranches(uri)).sort((b1, b2) => {
       if (b1 === defaultBranch) {
         return -1;
       } else if (b2 === defaultBranch) {
         return 1;
       }
       return b1.localeCompare(b2);
-    });
+    }).filter(branch => branch !== currentBranch);
```

Comment:
Filters the branches so that you can't create a PR to merge into the same branch!